src/gen_apple/ga_dev_xbee.h

```diff
@@ -1,20 +1,48 @@
 #include <Arduino.h>
-#include <XBee.h>
+//#include <XBee.h>
```

Comment:
If you're going to keep this in, it would be good to add a comment to explain why you removed this.